test.js

```diff
@@ -1,5 +1,5 @@
 var test = require('tape');
-var nodeCrypto  = require('./browser');
+var nodeCrypto  = require('./');
```

Comment:
No I want to test node against our code but I want it to also work in older node which doesn't support ecdsa